client/src/index.css

```diff
@@ -26,13 +26,6 @@ a:hover {
   color: #535bf2;
 }
 
-body {
```

Comment:
this set up shifted the whole body element? and not filling up the whole screen when expanding/shrinking the screen size, so I deleted them for now